web-console/knikubevirt/vm-details/vm-details.md

```diff
@@ -0,0 +1,114 @@
+# VM Details
+
+
+
+## Overview Tab
+
+![VM-overview](img/1-0-0.jpg)
+The user clicks a VM’s name from the list view and is taken to that VM’s Overview tab.
+The user sees detailed information about the VM’s configuration.
+
+
+## YAML
+
+Should be inline with the rest of Openshift. Can be seen [here](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/code-editor-updates/code-editor-updates)
+
+
+## Network Interfaces
+
+![VM-network intefaces - add NIC](img/2-0-0.jpg)
+This is Network Interfaces tab.
+
+
+![VM - add NIC modal](img/2-0-1.jpg)
+Default modal. The user is notified that activating this upcoming NIC requires a VM restart. 
+
+![VM - add NIC modal - filled in](img/2-0-2.jpg)
+User can choose between adding NIC only, or add NIC and restart VM to make it active.
+
+![VM - new NIC](img/2-0-3.jpg)
+The new NIC should always appear at the top of the list until the user restarts it. The new nic will have a'Pending VM restart' Link state (as a status).
+
+![VM - New NIC - popover](img/2-0-4.jpg)
+Hover on that link state (status) will display more details with a 'Restart VM action in a popover.
+
+![VM - Pending changes notification - view changes](img/2-0-5.jpg)
+Whenever a NIC has been added or modified in a way that requires the VM to be restarted, an inline notification should appear below the tab area reminding the user to do so. This notification should persist across all tab views, including the Overview (shown previously).
+
+![VM - Pending changes notification - view changes modal](img/2-0-6.jpg)
+Changes pending VM restart can be viewed in a dedicated modal.
+
+### Edit NIC
+
+![VM - NIC item action options](img/2-1-0.jpg)
```

Comment:
The required `Name` field isn't filled in here but the buttons to save are active. I assume the Name should be `Nic1` here.